lib/evaluateApplicants.ts

```diff
@@ -1,7 +1,7 @@
 import type { Record as AirtableRecord } from '@airtable/blocks/models';
 import type { Preset } from './preset';
 import type { Prompt } from './getChatCompletion';
-import { getChatCompletion } from './getChatCompletion/openai';
+import { getChatCompletion } from './getChatCompletion';
```

Comment:
nice, though in the future I could see it being better to just introduce a new layer of abstraction and use some library that can seamlessly interface with both APIs, and maybe includes [MCP support](https://modelcontextprotocol.io/introduction)

Reply:
(fine for now)